VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -264,4 +265,17 @@ public void onSuccess(CourseTopics courseTopics) {
         getTopicListTask.execute();
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        if (getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE) {
```

Comment:
Looking at it again, we can totally move-refactor this blurb 😉 into a util class.